app/models/eligibility_check.rb

```diff
@@ -22,6 +23,10 @@ class EligibilityCheck < ApplicationRecord
   belongs_to :region, optional: true
   has_one :application
 
+  enum :work_experience,
+       %i[under_9_months 9_to_20_months over_20_months],
```

Comment:
Should we store this as a string as we do elsewhere?

Reply:
I've amended the commit so they're stored as strings now, at the time I was thinking there was no major benefit to storing them as strings and it makes the database bigger, but I've just thought we probably want them as strings in analytics so makes sense, and it matches our other enums.